packages/react-scripts/template/README.md

```diff
@@ -566,6 +566,7 @@ Import Bootstrap CSS and optionally Bootstrap theme CSS in the ```src/index.js``
 import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap/dist/css/bootstrap-theme.css';
 ```
+Ensure the bootstrap css is imported before the rest of your app (i.e. before `import Routes from './routes';`) so that your styles are loaded afterwards.
```

Comment:
nit: Bootstrap / CSS

I'm not sure if I'm super-crazy about this wording, but I can't think of anything immediately better. I wonder if we should explain in more technical detail how overriding CSS works. 🤔